test/test_configuration.py

```diff
@@ -300,33 +300,8 @@ def test_cts_hashv_cwtclaims_payload_with_policy(
         with open("test/payloads/cts-hashv-cwtclaims-b64url.cose", "rb") as f:
             cts_hashv_cwtclaims = f.read()
 
-        submission = client.submit_claim_and_confirm(
-            cts_hashv_cwtclaims, receipt_type=ReceiptType.EMBEDDED
-        )
+        statement = client.submit_and_confirm(cts_hashv_cwtclaims).receipt_bytes
 
-        # store submission receipt in random file in tmp_path
+        # store statement
         receipt_path = tmp_path / "receipt.cose"
-        receipt_path.write_bytes(submission.receipt_bytes)
-        # print to preview what was accepted and to check if pretty-receipt understands the given receipt
-        main(["pretty-receipt", str(receipt_path)])
```

Comment:
`pretty-receipt` must always work, that is why it is included here. Please reinstate

Reply:
Once more, this a PR to a dev branch, some items will happen in later PRs.